frontend/src/app/page.tsx

```
@@ -1,12 +1,171 @@
+
+    return false;
+  }
+
+  async function fetchPersonalizedContent(): Promise<boolean> {
+    try {
+      const personalizedResponse = await axios.get(
+        `${apiURL}/content/feed/${userUID}`,
+        { timeout: 5000 }
+      );
+
+      if (personalizedResponse.data && personalizedResponse.data.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)